hazelcast/include/hazelcast/client/serialization/pimpl/SerializationConstants.h

```diff
@@ -62,6 +62,13 @@ namespace hazelcast {
                         CONSTANT_TYPE_FLOAT_ARRAY = -18,
                         CONSTANT_TYPE_DOUBLE_ARRAY = -19,
                         CONSTANT_TYPE_STRING_ARRAY = -20,
+
+                        // ------------------------------------------------------------
+                        // LANGUAGE SPECIFIC SERIALIZERS
+                        // USED BY CLIENTS (Not deserialized by server)
```

Comment:
Why do we have this comment? It seems wrong. Json is deserialized by the server.

Reply:
I copied the comment from here Javadoc https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/SerializationConstants.java#L94
I believe may be they wanted to say that it is not deserialized if no query, etc. But I agree it is better to remove it. I will remove it for C++ but may be Java side needs to be corrected as well, right?

Reply:
Yes, java side also needs to be updated.